src/components/Card.js

```diff
@@ -1,59 +1,77 @@
+import api from "../utils/Api.js";
```

Comment:
`import api from "../utils/Api.js";`

Please, delete it, and pass it into the constructor of `Card` from `index.js` if you want to use it inside the class


From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.